### PR TITLE
391 adjust chart settings revert

### PR DESCRIPTION
**Pull request by KaylaKremer**
This will become the new branch we use for #391 !!!!

http://alpha.blueraster.io/gfw-mapbuilder/391-adjust-chart-settings-revert/

### CODE REVIEW

**Comment by SwampGuzzler** [comment]
@KaylaKremer We need a loader to take up the space of the chart on `re-run`!

**Comment by SwampGuzzler** [comment]
@KaylaKremer If an `analysisModule` has no params, should it even have a **Gear** that toggles on that module's settings?

**Comment by SwampGuzzler** [comment]
Screen Shot 2019-07-30 at 2 24 45 PM
@KaylaKremer Instead of taking the `ReportSettings.js` component OUT of the DOM entirely when it is not displayed, can we instead just throw a `hidden` class on there. That way we won't setState on an unmounted component And we won't lose the UI settings adjustments the user has already made!

**Comment by SwampGuzzler** [comment]
https://github.com/wri/gfw-mapbuilder/issues/391#issuecomment-516538562

**Comment by KaylaKremer** [comment]
@SwampGuzzler Everything has been fixed. This PR should be ready to review and merge in now!

**Comment by KaylaKremer** [comment]
@SwampGuzzler PR is ready to review again. Build is updated.

File: src/js/components/AnalysisPanel/VegaChart.js [hunk 1]
```diff
@@ -165,32 +197,49 @@ export default class VegaChart extends Component {
...
-              <div className="vega-chart-divider"></div>
-              <div className='vega-chart-menu' onClick={() => this.setState({showDownloadOptions: !showDownloadOptions})}>
-                <SVGIcon className="vega-chart-menu-icon" id={'icon-download-grey'} />
+            <div className={component === 'Report' ? 'vega-chart_download-container-report' : 'vega-chart_download-container'}>
+              {/* <h3 className="vega-chart-label">{results.data.attributes.name}</h3> */}
+              <h3 className="vega-chart-label">{reportLabel}</h3>
+              <div className='vega-chart-menu-container'>
+                {
+                  (
+                  analysisId === 'TC_LOSS' ||
```
Comment (SwampGuzzler) [inline src/js/components/AnalysisPanel/VegaChart.js]: @KaylaKremer Let's do this automatically by running some kind of `filter` on all of our analysisModules looking if they have some `uiModules` in their array!

Comment (KaylaKremer) [reply src/js/components/AnalysisPanel/VegaChart.js]: Made a PR to clean this up! https://github.com/wri/gfw-mapbuilder/pull/472

**Pull request merged**